go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ipfs/go-graphsync
 
-go 1.12
+go 1.13
```

Comment:
maybe time to get this repo onboard with unified ci :) it would take care of bumping this too

Reply:
yes agreed.